tests/libc/enc/enc.c

```diff
@@ -102,8 +102,10 @@ int run_test_helper(const char* test_name, libc_test_function_t test_function)
 #endif
 
     /* Disable Open Enclave debug malloc checks. */
+#if OE_USE_DEBUG_MALLOC
     extern bool oe_disable_debug_malloc_check;
```

Comment:
This line should be deleted and instead include <openenclave/internal/malloc.h> which has this line already.

Reply:
Updated, thanks!